replica/table.cc

```diff
@@ -1027,12 +1027,24 @@ void table::do_trigger_compaction() {
 }
 
 void table::trigger_offstrategy_compaction() {
+    (void)perform_offstrategy_compaction().then_wrapped([this] (future<bool> f) {
+        if (f.failed()) {
+            auto ex = f.get_exception();
+            tlogger.warn("Offstrategy compaction of {}.{} failed: {}, ignoring", schema()->ks_name(), schema()->cf_name(), ex);
+        }
+    });
```

Comment:
perhaps a comment explaining why it's ok to not wait for the future.

Reply:
OK, I'll add one (though it's the preexisting behavior as submit_offstrategy didn't return a future)

Reply:
Fixed in v2